formicarium/formicarium/DiffRobot.py

```diff
@@ -64,7 +76,7 @@ def CmdVelCallback(self, msg: Twist) -> None:
         self.vel_l = ((v - w * (L / 2)) / r)
         self.vel_r = ((v + w * (L / 2)) / r)
 
-    def Move(self) -> None:
+    def move(self) -> None:
         deltaTime = (time.get_ticks() - self.previousTime) / 1000
```

Comment:
Keep in mind the coding standard

```suggestion
        delta_time = (time.get_ticks() - self.previous_time) / 1000
```